models/fates/R/write.configs.FATES.R

```diff
@@ -116,59 +133,53 @@ write.config.FATES <- function(defaults, trait.values, settings, run.id){
      jobsh <- readLines(con=system.file("template.job", package = "PEcAn.FATES"), n=-1)
    }
    
- # create host specific setttings
-   hostsetup <- ""
-   if (!is.null(settings$model$prerun)) {
-     hostsetup <- paste(hostsetup, sep="\n", paste(settings$model$prerun, collapse="\n"))
-   }
-   if (!is.null(settings$host$prerun)) {
-     hostsetup <- paste(hostsetup, sep="\n", paste(settings$host$prerun, collapse="\n"))
-   }
+ # create host specific settings
+  # hostsetup <- ""
+  # if (!is.null(settings$model$prerun)) {
+  #   hostsetup <- paste(hostsetup, sep="\n", paste(settings$model$prerun, collapse="\n"))
+  # }
+  # if (!is.null(settings$host$prerun)) {
+  #   hostsetup <- paste(hostsetup, sep="\n", paste(settings$host$prerun, collapse="\n"))
+  # }
 
-   hostteardown <- ""
-   if (!is.null(settings$model$postrun)) {
-     hostteardown <- paste(hostteardown, sep="\n", paste(settings$model$postrun, collapse="\n"))
-   }
-   if (!is.null(settings$host$postrun)) {
-     hostteardown <- paste(hostteardown, sep="\n", paste(settings$host$postrun, collapse="\n"))
-   }
+  # hostteardown <- ""
+  # if (!is.null(settings$model$postrun)) {
+  #   hostteardown <- paste(hostteardown, sep="\n", paste(settings$model$postrun, collapse="\n"))
+  # }
+  # if (!is.null(settings$host$postrun)) {
+  #   hostteardown <- paste(hostteardown, sep="\n", paste(settings$host$postrun, collapse="\n"))
+  # }
```

Comment:
For all these large chunks of code that you've commented out, are these things that are no longer used by CLM-FATES?